src/ssl.c

```diff
@@ -8197,12 +8200,17 @@ int wolfSSL_CertManagerEnableCRL(WOLFSSL_CERT_MANAGER* cm, int options)
 {
     int ret = WOLFSSL_SUCCESS;
 
-    (void)options;
-
     WOLFSSL_ENTER("wolfSSL_CertManagerEnableCRL");
     if (cm == NULL)
         return BAD_FUNC_ARG;
 
+    if (options == 0) {
+        /* Turn off doing Leaf CRL check */
+        cm->crlEnabled = 0;
+        /* Turn off all checks */
+        cm->crlCheckAll = 0;
+    }
+
```

Comment:
This greatly changes how this API functions. Is it safe to change the default to being off? I think it would be better to make this conditional on a compile time macro.